services/QuillLMS/app/assets/stylesheets/pages/content_hub/subject_page.scss

```diff
@@ -188,4 +195,17 @@
       margin-bottom: 0px;
     }
   }
+  &.social-studies-subject-page {
+    header {
+      background-image: url("https://assets.quill.org/images/pages/social_studies_subject_page/header-backdrop.svg");
+    }
+  }
+  &.interdisciplinary-science-subject-page {
+    header {
+      background-image: url("https://assets.quill.org/images/pages/interdisciplinary_science_subject_page/header_backdrop.svg");
+    }
+    .partners {
+      border-top: 1px solid $quill-grey-15;
+    }
```

Comment:
No strong feelings about this so take or leave it, but I assume that this sub-selector is applied only to the science section because of the way these are listed next to each other.  I wonder if it'd make sense to do something more generic like adding a `:first-child` selector to the general `.partners` selector further up so that if/when we add more of these, or re-order them, we don't have to tweak the CSS for this.

Reply:
This is actually just a temporary measure while we wait to get permission to use some teacher quotes. In the final design, we won't have this line, because there will be a different block here. So I agree with your recommendation in general, but I think in this instance since it'll be removed soon I'll leave it as is.